### PR TITLE
Remove "Unicode" from build configuration names

### LINKED ISSUE #10114: Appveyor build issue with Vcpkg
I have noticed this
```
Vcpkg is unable to link because we cannot decide between Release and Debug libraries. Please define the property VcpkgConfiguration to be 'Release' or 'Debug'
```
https://ci.appveyor.com/project/donho/notepad-plus-plus/builds/39878689/job/s2667yupokml43gx#L38
https://ci.appveyor.com/project/donho/notepad-plus-plus/builds/39878689/job/s2667yupokml43gx#L524

### PR DESCRIPTION
**Pull request by mere-human**
Rename "Unicode Debug" => "Debug", "Unicode Release" => "Release"
since there is no non-Unicode version.
Also, fix the name in other configs for Git, AppVeyor and in the docs.

Fix #10114

### CODE REVIEW

**Comment by donho** [comment]
I build both x86 & x64 under VS2019, both got the error with "Rebuild" command: 

image

image

I don't get how my environment is different from AppVeyer's one.

**Comment by donho** [comment]
@mere-human 

Please ignore above message. For the unknown reason, VS screwed up the Scintilla project file after merging your PR :

image

After restoring it, the build works with no issue.

**Comment by mere-human** [comment]
> I build both x86 & x64 under VS2019, both got the error with "Rebuild" command:

Maybe, some leftovers from the old build cause such problem.
Did you rebuild project or solution? I tried both, but I don't see such error.
I built all the configurations locally, did a quick open/save scenarios, and it was fine.

**Comment by mere-human** [comment]
> Please ignore above message. For the unknown reason, VS screwed up the Scintilla project file after merging your PR :

Oh, I see.
Sometimes Visual Studio asks if I want to upgrade the solution. Maybe that's why.

**Comment by donho** [comment]
> the unknown reason, VS screwed up the Scintilla project file after merging your PR 

The unknown reason should be: I've launch `SciLexer.vcxproj` separately then close it before reload the solution of your PR.

File: PowerEditor/visual.net/notepadPlus.sln [hunk 1]
```diff
@@ -8,42 +8,42 @@ Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "Notepad++", "notepadPlus.vc
...
+		{FCF60E65-1B78-4D1D-AB59-4FC00AC8C248}.Debug|ARM64.ActiveCfg = Debug|ARM64
+		{FCF60E65-1B78-4D1D-AB59-4FC00AC8C248}.Debug|ARM64.Build.0 = Debug|ARM64
+		{FCF60E65-1B78-4D1D-AB59-4FC00AC8C248}.Debug|Win32.ActiveCfg = Debug|Win32
+		{FCF60E65-1B78-4D1D-AB59-4FC00AC8C248}.Debug|Win32.Build.0 = Debug|Win32
+		{FCF60E65-1B78-4D1D-AB59-4FC00AC8C248}.Debug|x64.ActiveCfg = Debug|x64
+		{FCF60E65-1B78-4D1D-AB59-4FC00AC8C248}.Debug|x64.Build.0 = Debug|x64
+		{FCF60E65-1B78-4D1D-AB59-4FC00AC8C248}.Release|ARM64.ActiveCfg = Release|ARM64
+		{FCF60E65-1B78-4D1D-AB59-4FC00AC8C248}.Release|ARM64.Build.0 = Release|ARM64
+		{FCF60E65-1B78-4D1D-AB59-4FC00AC8C248}.Release|Win32.ActiveCfg = Debug|Win32
+		{FCF60E65-1B78-4D1D-AB59-4FC00AC8C248}.Release|Win32.Build.0 = Debug|Win32
```
Comment (ozone10) [inline PowerEditor/visual.net/notepadPlus.sln:32]: I think these should be `Release`

File: PowerEditor/visual.net/notepadPlus.sln [hunk 2]
```diff
@@ -8,42 +8,42 @@ Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "Notepad++", "notepadPlus.vc
...
+		{FBE04237-9C7B-4973-9C60-505975998B39}.Debug|ARM64.ActiveCfg = Debug|ARM64
+		{FBE04237-9C7B-4973-9C60-505975998B39}.Debug|ARM64.Build.0 = Debug|ARM64
+		{FBE04237-9C7B-4973-9C60-505975998B39}.Debug|Win32.ActiveCfg = Debug|Win32
+		{FBE04237-9C7B-4973-9C60-505975998B39}.Debug|Win32.Build.0 = Debug|Win32
+		{FBE04237-9C7B-4973-9C60-505975998B39}.Debug|x64.ActiveCfg = Debug|x64
+		{FBE04237-9C7B-4973-9C60-505975998B39}.Debug|x64.Build.0 = Debug|x64
+		{FBE04237-9C7B-4973-9C60-505975998B39}.Release|ARM64.ActiveCfg = Release|ARM64
+		{FBE04237-9C7B-4973-9C60-505975998B39}.Release|ARM64.Build.0 = Release|ARM64
+		{FBE04237-9C7B-4973-9C60-505975998B39}.Release|Win32.ActiveCfg = Debug|Win32
+		{FBE04237-9C7B-4973-9C60-505975998B39}.Release|Win32.Build.0 = Debug|Win32
```
Comment (ozone10) [inline PowerEditor/visual.net/notepadPlus.sln:44]: I think these should be `Release`

Comment (mere-human) [reply PowerEditor/visual.net/notepadPlus.sln]: Done.

**Pull request closed without merging**